packages/viewer/src/interactions/multiPointerInteractionHandler.ts

```diff
@@ -0,0 +1,65 @@
+import { Point } from '@vertexvis/geometry';
+import { InteractionApi } from './interactionApi';
+import { MultiTouchInteractionHandler } from './multiTouchInteractionHandler';
+
+export class MultiPointerInteractionHandler extends MultiTouchInteractionHandler {
+  private touchPoints: Record<string, Point.Point> = {};
+
+  public constructor() {
+    super();
+    this.handlePointerDown = this.handlePointerDown.bind(this);
+    this.handlePointerMove = this.handlePointerMove.bind(this);
+    this.handlePointerUp = this.handlePointerUp.bind(this);
+  }
+
+  public dispose(): void {
+    this.element?.removeEventListener('pointerdown', this.handlePointerDown);
+    super.dispose();
+  }
+
+  public initialize(element: HTMLElement, api: InteractionApi): void {
+    super.initialize(element, api);
+
+    element.addEventListener('pointerdown', this.handlePointerDown);
+  }
+
+  private handlePointerDown(event: PointerEvent): void {
+    event.preventDefault();
+    const point = Point.create(event.screenX, event.screenY);
+    this.touchPoints = {
+      ...this.touchPoints,
+      [event.pointerId]: point,
+    };
+
+    window.addEventListener('pointermove', this.handlePointerMove);
+    window.addEventListener('pointerup', this.handlePointerUp);
```

Comment:
I suspect there's a similar bug with adding/removing event listeners. Should add checks so we only register event listeners on the first pointer down, and deregister on the last point up.